pavelib/prereqs.py

```diff
@@ -239,6 +246,16 @@ def package_in_frozen(package_name, frozen_output):
 
 @task
 @timed
+def install_coverage_prereqs():
+    """ Install python prereqs for measuring coverage. """
+    if no_prereq_install():
+        print NO_PREREQ_MESSAGE
+        return
```

Comment:
It needs to be at the task level not in an inner method because otherwise in the usual install_python_prereqs use case the message will be printed out for each of the requirement files (pre, github, base, etc.)